src/value.rs

```diff
@@ -321,15 +319,18 @@ mod tests {
                     ''${ interpolation was escaped }
                     two single quotes: '''
                     three single quotes: ''''
-                "#.replace("|trailing-whitespace", "").as_str()
-            )),
-            vec![
-                StrPart::Literal(String::from(
-                    // Get the below with nix repl
-                    "    \n          \nThis is a multiline string :D\n  indented by two\n\\'\\'\\'\\'\\\n${ interpolation was escaped }\ntwo single quotes: ''\nthree single quotes: '''\n"
-                ))
-            ]
-        );
+                "#
+            .replace("|trailing-whitespace", "")
+            .as_str(),
+        ))[..]
+        {
+            assert_eq!(lit,
+                // Get the below with nix repl
+                "    \n          \nThis is a multiline string :D\n  indented by two\n\\'\\'\\'\\'\\\n${ interpolation was escaped }\ntwo single quotes: ''\nthree single quotes: '''\n"
```

Comment:
Given above's change we should also add a test-case for interpolation-nodes.

Reply:
I agree, but I don't know what exactly we should test.

Reply:
I'd suggest to test the result of a string with an actual interpolation. And perhaps also include something like your code-snippet from #66 in a test :)